src/core/pointcloud/qgspointcloudstatscalculationtask.cpp

```diff
@@ -40,7 +43,7 @@ void QgsPointCloudStatsCalculationTask::cancel()
 {
   QgsTask::cancel();
   mFeedback->cancel();
-  emit QgsTask::taskTerminated();
+  //emit QgsTask::taskTerminated();
```

Comment:
```suggestion
```

Reply:
Done, squashed and rebased against current master